src/InstrumentationEngine.Installer.NuGet/Module/InstrumentationEngine.Module.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<package>
+  <metadata>
+    <id>$packageid$</id>
+    <version>$version$</version>
+    <title>Microsoft Instrumentation Engine Module</title>
+    <authors>$authors$</authors>
+    <owners>$authors$</owners>
+    <projectUrl>https://github.com/Microsoft/CLRInstrumentationEngine</projectUrl>
+    <license type="file">$licenseFile$</license>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Microsoft Instrumentation Engine Package containing the MergeModule</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+  </metadata>
+  <files>
+    <file src="$bindir$\InstrumentationEngine.Module_x64.$version$$msisuffix$.msm" target="Tools" />
```

Comment:
Is tools the right target here? Perhaps just content?

Reply:
From [https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#from-a-convention-based-working-directory](https://docs.microsoft.com/en-us/nuget/create-packages/creating-a-package#from-a-convention-based-working-directory)

It does seem like Content is the right folder for this.

---
In reply to: [354496267](https://github.com/microsoft/CLRInstrumentationEngine/pull/178#discussion_r354496267) [](ancestors = 354496267)